tests/functional/base.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (C) 2017  Custodia Project Contributors - see LICENSE file
+from __future__ import absolute_import
+
+import os
+import shutil
+import socket
+import subprocess
+import sys
+import time
+from string import Template
+
+import pytest
+
+from custodia.client import CustodiaHTTPClient
+from custodia.compat import url_escape
+from custodia.server.args import parse_args
+from custodia.server.config import parse_config
+
+
+class CustodiaServerRunner(object):
```

Comment:
Please add ``@pytest.mark.servertest`` so we can skip the function tests together with ``test_cli``